scipy/stats/_survival.py

```diff
@@ -0,0 +1,116 @@
+from dataclasses import make_dataclass
+import numpy as np
+from ._censored_data import CensoredData
+
+
+__all__ = ['ecdf']
+
+
+ECDFResult = make_dataclass('ECDFResult', ['x', 'cdf', 'sf'])
```

Comment:
Can we instead declare classes. The issue with doing it like that is that IDEs (also iPython) are not smart enough to follow through and provide autocomplete.